pkgs/data/fonts/rounded-mgenplus/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, p7zip }:
+
+let
+  pname = "rounded-mgenplus";
+  version = "2015-06-02";
```

Comment:
You could also call the version `"20150602"` here and get rid of the `replaceStrings` below. Both date formats are acceptable and used by many packages. Your choice.